dsc/workflows/base/__init__.py

```diff
@@ -396,6 +397,8 @@ def item_submissions_iter(self) -> Iterator[ItemSubmission]:
             dspace_metadata = self.create_dspace_metadata(item_metadata)
```

Comment:
One challenge we will face with moving the DSpace metadata functionality into `ItemSubmission` is that the `metadata_mapping` attribute currently lives on the `Workflow` class, we can figure out how to address that during the `submit` ticket

Reply:
Hmm, is there an issue with moving the `create_dspace_metadata` functionality to the `ItemSubmission` class? If we move that method to `ItemSubmission`, we can add a parameter for the `metadata_mapping_path`, which the calling context (i.e., the `Workflow` class) can pass to the method call. 🤔